patternfly/patternfly-elements
# fix: #1490: Make pfe-icon set CSS properties instead of overriding local var

Overriding local vars is overriding author's intentions in some cases, making pfe-icon set the CSS properties the vars effect instead of overriding the vars



## <Pull request description>


### Related issues

- (#1490) PFE-Icon is setting local variables overriding author's intentions


### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Icon demo](https://deploy-preview-1491--happy-galileo-ea79c4.netlify.app/elements/pfe-icon/demo/) 


### Testing instructions

1. Go to pfe-icon demo page and make sure nothing has changed, but setting --pfe-icon-Color should change icon color in all circumstances.


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
